Lib/test/test_readline.py

```diff
@@ -210,8 +210,10 @@ def display(substitution, matches, longest_match_length):
         self.assertIn(b"result " + expected + b"\r\n", output)
         self.assertIn(b"history " + expected + b"\r\n", output)
 
-    @unittest.skipIf(is_editline,
-                     "editline history size configuration is broken")
+    # History size was added in readline 6.0
+    # See https://cnswww.cns.cwru.edu/php/chet/readline/CHANGES
+    @unittest.skipIf(getattr(readline, "_READLINE_VERSION", 0x00) < 0x600,
```

Comment:
We've been using _READLINE_VERSION without a fallback value in test_init in line 116 for a while and I don't remember any issue reported on bugs.p.o because of it. I think we can safely drop the fallback value.